package.json

```diff
@@ -54,7 +53,9 @@
     "peerDependencyRules": {
       "allowedVersions": {
         "cypress-image-snapshot>cypress": "*",
-        "jest-image-snapshot>jest": "*"
+        "jest-image-snapshot>jest": "*",
+        "@phenomnomnominal/tsquery>typescript": "5.x",
+        "eslint-plugin-etc>typescript": "5.x"
```

Comment:
Fixing some peer deps warnings I hadn't fixed when upgrading `eslint-config-galex`. After I asked for it, pnpm added a CLI flag to check for peer deps issues: `pnpm i --resolution-only`, which is quite convenient.